src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -109,6 +115,10 @@ protected function createEntityManagerMock($repositoryMock)
         $classMetadata = $this->createMock(
             class_exists(ClassMetadataInfo::class) ? ClassMetadataInfo::class : ClassMetadata::class
         );
+        $classMetadata
+            ->method('getName')
+            ->willReturn($repositoryMock->getClassName())
+        ;
```

Comment:
This change does not seem to be needed.

Reply:
@xabbuh Actually, this is need to pass lowest deps: https://github.com/HypeMC/symfony/actions/runs/9289115507/job/25562199480 . The reason it fails is because of the following logic: https://github.com/symfony/symfony/blob/31c597fe3ed692dcbd8dbe2f6e2af63cd4a1c679/src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php#L109-L111

The `ClassMetadataInfo::getName()` method [doesn't have a return type](https://github.com/doctrine/orm/blob/4a01a76a17f10851393db48efddcbf3377938a3d/src/Mapping/ClassMetadataInfo.php#L3575-L3578) so `null` is returned, while `ClassMetadata::getName()` [does](https://github.com/doctrine/orm/blob/9d4f54b9a476f13479c3845350b12c466873fc42/src/Mapping/ClassMetadata.php#L2456-L2459), so an empty string is returned which doesn't cause an error.